components/integrations/installed_outgoing_webhook.tsx

```diff
@@ -159,25 +177,20 @@ export default class InstalledOutgoingWebhook extends React.PureComponent<Props>
 
         let actions = null;
         if (this.props.canChange) {
-            actions = (
-                <div className='item-actions'>
+            actions =
+                (<div className='item-actions'>
```

Comment:
nit: different coding style as before, but still not what we do unfortunately:

```suggestion
            actions = (
                <div className='item-actions'>
```